tools/image-info

```diff
@@ -75,8 +75,14 @@ def parse_unit_files(s, expected_state):
 
 
 def subprocess_check_output(argv, parse_fn=None):
-    output = subprocess.check_output(argv, encoding="utf-8")
-    return parse_fn(output) if parse_fn else output
+    sp = subprocess.run(argv, encoding="utf-8", stdout=subprocess.PIPE)
```

Comment:
Pylint usually fails when `check` is not passed. We're not running pylint here, but I think it's good form to pass it.